tools/check_format.py

```diff
@@ -143,6 +144,32 @@ def checkNamespace(file_path):
   return []
 
 
+def fixJavaProtoOptions(file_path):
+  java_multiple_files = False
+  java_package_correct = False
+  package_name = None
+  for line in fileinput.FileInput(file_path):
+    if line.startswith("package "):
+      package_name = PROTO_PACKAGE_REGEX.search(line).group(1)
```

Comment:
do you need to check the cardinality of the search return before pulling out the 1th match?

Reply:
s/PROTO_PACKAGE_REGEX.search(line)/result/